.github/workflows/generate-token-lists.yml

```diff
@@ -151,6 +151,13 @@ jobs:
             version: true
             command: yarn arbify --l2NetworkID 421614 --prevArbifiedList https://tokenlist.arbitrum.io/ArbTokenLists/421614_arbed_coingecko.json  --tokenList https://tokens.coingecko.com/uniswap/all.json --newArbifiedList ./src/ArbTokenLists/421614_arbed_coingecko.json
 
+          # Orbit Chains
+          - name: Xai Arbify Uniswap
+            paths:
+              - ArbTokenLists/660279_arbed_uniswap_labs.json
+            version: true
+            command: yarn arbify --l2NetworkID 660279 --ignore-previous-list --tokenList https://gateway.ipfs.io/ipns/tokens.uniswap.org --newArbifiedList ./src/ArbTokenLists/660279_arbed_uniswap_labs.json
```

Comment:
`--ignore-previous-list` will be replaced with `--prevArbifiedList` in a subsequent PR, once the list is uploaded

Reply:
https://github.com/OffchainLabs/arbitrum-token-lists/pull/122